tests/should_succeed/add_sub_mul_div_mod.jou

```diff
@@ -38,9 +38,11 @@ def main() -> int:
     printf("%d\n", 'a'+'a'+'a') # Output: 35
 
     # Integer overflow. Not UB in Jou.
+    printf("%hd\n", 20000 + 20000) # Output: -25536
```

Comment:
This does the addition with `int`s, and then passes an `int` to the `printf()` function when it is expecting a float. Let's instead add `short`s:

```suggestion
    printf("%hd\n", 20000s + 20000s) # Output: -25536
```

Reply:
- **Short literals** are just like in literals, except that they have an extra `S` at the end (e.g. `114S`),
    The `S` must be uppercase.
```suggestion
    printf("%hd\n", 20000S + 20000S) # Output: -25536
```